<?xml version="1.0"?>

<launch>

  <arg name="sensor_id" default="imu_1" />
  <arg name="sensor_number" default="1" />
  <arg name="topic_name" default="/imu/data" />

  <node pkg="imu_ros" type="imu_cusum" respawn="false" name="imu_detector" output="screen">
    <param name="input_topic" type="string" value="$(arg topic_name)"/>
  </node>
</launch>
